src/locales/en.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
The i18n multi-language json file is not useful and should be deleted.

Reply:
Done